src/emulator/io/include/io/functions.h

```diff
@@ -35,7 +36,7 @@ int create_dir(const char *dir, int mode, const char *pref_path);
 int remove_file(const char *file, const char *pref_path);
 int create_dir(const char *dir, int mode, const char *pref_path);
 int remove_dir(const char *dir, const char *pref_path);
-int stat_file(const char *file, SceIoStat *stat, const char *pref_path);
+int stat_file(const char *file, SceIoStat *stat, const char *pref_path, HostState &host);
```

Comment:
Possible to pass IOState instead?

Reply:
That's what i did initially but I need `host.kernel.base_tick`.
I guess I can just pass _that_ (gonna be an extra argument though), I'll see.